src/main/java/frc/robot/Robot.java

```diff
@@ -51,8 +52,9 @@ public void teleopInit() {
 
     @Override
     public void robotPeriodic() {
-        RoborioUtils.updateRioUtils(); // Better to be first
+        CycleTimeUtils.updateCycleTime(); // Better to be first
         CommandScheduler.getInstance().run();
```

Comment:
think about adding in cycle time util something that measure "CommandScheduler.getInstance().run();" time